Makefile

```diff
@@ -0,0 +1,11 @@
+# $FreeBSD$
+
+TESTSDIR=	${LOCALBASE}/tests/tcptestsuite
+ATF_TESTS_SH=	tcptestsuite_atf_test
```

Comment:
The name is pretty redundant.  "tcptestsuite" is the name of the parent directory, and "atf_test" is the file type.  But to me, nothing better comes to mind.  I won't object if you don't change this.

Reply:
Yeah, it's a bit redundant, but my language center has reached its quota for today.  :)